RELEASE-NOTES.txt

```diff
@@ -1,5 +1,6 @@
 14.4
 -----
+* Post Settings: Fixes the display of posts which are to be immediately published.
```

Comment:
Maybe this can be a bit more specific to the date? Something like:
`Post Settings: Fixes the displayed publish date of posts which are to be immediately published.`

Reply:
Much better! 👍